ebusd/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## TBC
+
+- BREAKING: Remove old style loglevel and logareas option
```

Comment:
This allows the user to select log levels for each individual area.  The previous config only allowed you to select the default log level and change the log level for one of the 4 other areas.

Reply:
great! I was too lazy to add all the individual log levels. thanks for this!